### ISSUE TITLE
potential $@ quoting issues

### ISSUE THREAD

**Issue by cvrebert** [saltstack/salt-bootstrap#356]
```
In bootstrap-salt.sh line 583:
    echo $@ | sed "s/^\([\"']\)\(.*\)\1\$/\2/g"
         ^-- SC2068: Add double quotes around ${@}, otherwise it's just like $* and breaks on spaces.

In bootstrap-salt.sh line 591:
    echo $@ | sed -r 's/([^A-Z-])([A-Z])/\1 \2/g'
         ^-- SC2068: Add double quotes around ${@}, otherwise it's just like $* and breaks on spaces.

In bootstrap-salt.sh line 599:
    echo $@ | tr -s '[:space:]' '\n' | awk '!x[$0]++'
         ^-- SC2068: Add double quotes around ${@}, otherwise it's just like $* and breaks on spaces.

In bootstrap-salt.sh line 614:
    for release_file in $(echo $@ | sed -r 's:[[:space:]]:\n:g' | sort --unique --ignore-case); do
                               ^-- SC2068: Add double quotes around ${@}, otherwise it's just like $* and breaks on spaces.

In bootstrap-salt.sh line 1047:
    apt-get install -y -o DPkg::Options::=--force-confold $@; return $?
                                                          ^-- SC2068: Add double quotes around ${@}, otherwise it's just like $* and breaks on spaces.

In bootstrap-salt.sh line 1056:
    apt-get upgrade -y -o DPkg::Options::=--force-confold $@; return $?
                                                          ^-- SC2068: Add double quotes around ${@}, otherwise it's just like $* and breaks on spaces.

In bootstrap-salt.sh line 3694:
        emerge --autounmask-write --getbinpkg $@; return $?
                                              ^-- SC2068: Add double quotes around ${@}, otherwise it's just like $* and breaks on spaces.

In bootstrap-salt.sh line 3696:
    emerge --autounmask-write $@; return $?
                              ^-- SC2068: Add double quotes around ${@}, otherwise it's just like $* and breaks on spaces.
```

**Comment by cvrebert** [comment]
Apparently fixed in #358.

**Comment by s0undt3ch** [comment]
Not all of them :)
